schema/tables/power_sensors.yml

```diff
@@ -0,0 +1,10 @@
+name: power_sensors
+notes: Returns useful results on Intel Macs only.
+examples: >-
+  See the total power usage of an Intel Mac.
```

Comment:
If I see high power usage or low power usage, is that bad? Why? 

How do I determine if something is high or low?